packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -533,7 +542,7 @@ class BottomSheet extends Component {
 					keyboardVerticalOffset={ -safeAreaBottomInset }
 				>
 					<View onLayout={ this.onHeaderLayout }>
-						{ ! ( Platform.OS === 'android' && isFullScreen ) && (
+						{ ( ! isFullScreen || showDragIndicator ) && (
```

Comment:
This is a bit of a risky change. By using a prop, it's now on the parent component to make sure the drag indicator is not shown. Currently it's hidden on Android when fullscreen. 
I looked for any cases where the `BottomSheet` is set fullscreen and I couldn't find any. There are some cases where a bottom sheet is fullscreen , link management for example, but those are using bottom sheet sub components. 

I'm open to returning the platform check and making the prop "opt in" but I felt the added complexity was unnecessary.